Schema based on diagram.sql

```diff
@@ -0,0 +1,58 @@
+/* Database schema to keep the structure of entire database. */
+CREATE TABLE "patients" (
+  "id" int,
+  "name" varchar,
+  "date_of_birth" date,
+  PRIMARY KEY ("id")
+);
+
+CREATE TABLE "medical_histories" (
+  "id" int,
+  "admitted_at" timestamp,
+  "patient_id" int REFERENCES patients("id"), -- Specify the referenced column
+  "status" varchar,
+  PRIMARY KEY ("id"),
+  CONSTRAINT "FK_medical_histories_patient_id" FOREIGN KEY ("patient_id") REFERENCES "patients"("id") -- Corrected constraint name and syntax
+);
+
+CREATE INDEX ON "medical_histories" (patient_id);
+
+CREATE TABLE "treatments" (
+  "id" int,
+  "type" varchar,
+  "name" varchar,
+  PRIMARY KEY ("id")
+);
+
+CREATE TABLE "invoices" (
+  "id" int,
+  "total_amount" decimal,
+  "generated_at" timestamp,
+  "payed_at" timestamp,
+  "medical_history_id" int,
```

Comment:
- You have done great so far :+1: :100: But there are a few changes you need to make before moving to the next requirement, :+1: 
- [ ] Looking at the diagram, `medical_history_id` is a foreign key and therefore should have a defined reference to the `Medical histories` table. and also it should be indexed. I'll suggest you update this. :+1: :+1: